app/src/androidTest/java/com/litewallet/analytics/ConstantsTests.java

```diff
@@ -0,0 +1,79 @@
+package com.litewallet.analytics;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import android.util.Log;
+import androidx.test.ext.junit.rules.ActivityScenarioRule;
+
+import com.breadwallet.presenter.activities.intro.IntroActivity;
+import com.breadwallet.tools.util.BRConstants;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.net.URI;
+
+@RunWith(AndroidJUnit4.class)
+public class ConstantsTests {
```

Comment:
since this is not UI test, we can move it to `app/src/test` instead of `app/src/androidTest`.
probably I will help to setup the testing stuff on the next PR